### ISSUE TITLE
RedisAI throws error when it finds len() while torch script wont.

### ISSUE THREAD

**Issue by hhsecond** [RedisAI/RedisAI#106]
RedisAI SCRIPTSET throws error when it finds `len()` in the script while PyTorch's torch script won't. Checking the length is a useful thing to have. Is this something we can't have without much hustle? Below is an error trace from RedisAI
```
ERR: 
undefined value len:
        max_detections = []
        should_break = False
        while detections_class.size(0) != 0 and not should_break:
            max_detections.append(detections_class[0].unsqueeze(0))
            if detections_class.size(0) == 1:
                should_break = True
            else:
                ious = bbox_iou(max_detections[-1], detections_class[1:], True)
                detections_class = detections_class[1:][ious < nms_thresh]
        if len(max_detections) > 0:
           ~~~ <--- HERE
            output = torch.cat(max_detections)
    return output
```

**Comment by lantiga** [comment]
This may have something to do with the version of `libtorch` we use. Are you sure you're using the same version on either sides?

**Comment by hhsecond** [comment]
I couldn't find a difference in the version I am using. I checked the CPP source and it couldn't be anything else. Perhaps the binary `get_deps.sh` (1.0.1) is pulling is different than `pip install https://download.pytorch.org/whl/cpu/torch-1.0.1-cp37-cp37m-linux_x86_64.whl`. I'll try to reproduce it later today and verify again

**Comment by itamarhaber** [comment]
Could this be the root cause https://github.com/pytorch/pytorch/issues/13376? Anyway, also got hit by this.

**Comment by lantiga** [comment]
This will likely be solved when #115 is done.

**Comment by hhsecond** [comment]
This is fixed with #141
